test/unit/models/Detainees.test.js

```diff
@@ -1,7 +0,0 @@
-'use strict';
```

Comment:
you might like to check that `DetaineeEvents` get made with the right number of fixtures